content/pt-br/tightly_coupled_architectures.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Arquitetura Acoplada
+status: Completed
+category: Propriedade
```

Comment:
```suggestion
category: propriedade
```

Reply:
Hi, it's ok for me, but I repeated the capital letter in original text.